app/src/main/java/org/oppia/android/app/testing/DragDropTestFragmentPresenter.kt

```diff
@@ -0,0 +1,53 @@
+package org.oppia.android.app.testing
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import org.oppia.android.R
+import org.oppia.android.app.recyclerview.BindableAdapter
+import org.oppia.android.databinding.DragDropTestFragmentBinding
+import javax.inject.Inject
+
+class DragDropTestFragmentPresenter @Inject constructor(
+  private val fragment: Fragment,
+  private val singleTypeBuilderFactory: BindableAdapter.SingleTypeBuilder.Factory
+) {
+  var dataList = mutableListOf("Item 1", "Item 2", "Item 3", "Item 4")
```

Comment:
Does this need to be a var, or public? Prefer keeping things immutable (val, listOf), and private by default.